### DISCUSSION TITLE
an easy way to export lazer files

### DISCUSSION THREAD

**Discussion by shinmorisawa** [ppy/osu#30754] [Ideas] [answered]
i've been distro hopping lately, and i want to easily move my files without having to copy the osu directory, move it to my usb, and import it. maybe a single file that i can just drag onto the osu! window and all of my data gets put in the right place?

**Comment by peppy** [comment, marked as answer]
Please copy the files as you propose.

**Reply by shinmorisawa** [reply]
Ok then
